docs/host/host.md

```diff
@@ -25,6 +25,20 @@ For admin users, you can click **Enable Maintenance Mode** to evict all VMs from
 
 ![node-maintenance.png](/img/v1.2/host/node-maintenance.png)
 
+After a while the target node will enter maintenance mode successfully.
+
+![node-enter-maintenance-mode.png](/img/v1.3/troubleshooting/node-enter-maintenance-mode.png)
+
+:::info important
+
+Check those [known limitations and workarounds](../troubleshooting/host.md#an-enable-maintenance-mode-node-stucks-on-cordoned-state) before you click this menu or when you have encountered some issues.
+
+If you have attached any volume to this node manually, it may block the node maintenance, check the affection of [Manually Attached Volumes](../troubleshooting/host.md#manually-attached-volumes) and set a proper global option.
+
+If you have any single-replica volume, it may block the node maintenance, check the affection of [Single-Replica Volumes](../troubleshooting/host.md#single-replica-volumes) and set a proper global option.
```

Comment:
```suggestion
If you have any single-replica volume, it may block the node maintenance, check the section  [Single-Replica Volumes](../troubleshooting/host.md#single-replica-volumes) and set a proper global option.
```

Reply:
Thanks @ibrokethecloud, all related parts are updated.